lighthouse-core/gather/connections/connection.js

```diff
@@ -96,8 +96,8 @@ class Connection {
       this._callbacks.delete(object.id);
       if (object.error) {
         log.formatProtocol('method <= browser ERR',
-            {method: callback.method, params: object.result}, 'error');
-        callback.reject(object.result);
+            {method: callback.method}, 'error');
+        callback.reject(object.error);
```

Comment:
we might want to augment this message with information about it being from the debugging protocol (since I believe the log above that will only be on if you specify `--verbose`)